front/src/containers/PlayerPanel.js

```diff
@@ -48,13 +48,20 @@ const DisplayMetadata = (props) => {
 		artist: "UDONradio"
 	}
 	const {title, album, artist} = {...placeholder, ...props.lastSong};
-
-  return <span className="dynamic">
-		<b>{artist} - </b>{title}{album && <i> - {album}</i>}
-  </span>
+  const is_new = (props.lastSong) ? props.lastSong.new : false;
+  return <React.Fragment>
+    <Grid.Column>
```

Comment:
I am not sure a Grid.Column is necessary here.

Reply:
This component should be rendered conditionnaly when is_new is true. You can find an example of conditonal rendering at line 60